backend/database/actionsWithTables/categoryActions.js

```diff
@@ -0,0 +1,32 @@
+const { Category } = require('../tables/category');
+
+const addNewCategory = async (data) => {
+    return await Category.create({
+        name: data.name,
+    });
+};
+
+const returnCategorys = async () => {
+    const categorys = await Category.findAll({
+        attributes: ['category_id', 'name', 'is_deleted'],
+        where: {
+            is_deleted: false,
+        }
+    });
+    return await categorys;
+};
+
+const isDeleteCategory = async (data) => {
```

Comment:
Remove action function to as it is doing
```suggestion
const deleteCategory = async (data) => {
```